test/functional/lib/ipfs-path.test.js

```diff
@@ -1,10 +1,22 @@
 'use strict'
-const { describe, it } = require('mocha')
+const { describe, it, beforeEach } = require('mocha')
 const { expect } = require('chai')
-const { safeIpfsPath } = require('../../../add-on/src/lib/ipfs-path')
+const { URL } = require('url')
+const { safeIpfsPath, createIpfsPathValidator } = require('../../../add-on/src/lib/ipfs-path')
+const { initState } = require('../../../add-on/src/lib/state')
+const createDnsLink = require('../../../add-on/src/lib/dns-link')
+const { optionDefaults } = require('../../../add-on/src/lib/options')
 
 // https://github.com/ipfs/ipfs-companion/issues/303
-describe('IPFS Path Sanitization', function () {
+describe('ipfs-path.js', function () {
```

Comment:
I assume no reason other than legacy that fat-arrow functions aren't used everywhere here? or is it because of `mocha` patching or just a pattern maintained?

Reply:
I think it is just an unwritten convention so that we don't need to think where arrow function will be a problem for mocha: https://mochajs.org/#arrow-functions